docs/source/packs.rst

```diff
@@ -133,6 +133,10 @@ version or **upgrade to latest** if the version is not specified. Your config fi
 overwritten, so you can revert to an older version just as easily, but for production deployments
 we recommend to always specify versions in case there are major changes in ``latest``.
 
+.. note::
+
+    It is also possible to download and install packs from GitLab using a SSH based URL through the web interface in the Actions section.
```

Comment:
Suggestion that add "New feature! As of StackStorm 3.4." to the beginning of the note, like the other notes where features are only available in certain releases.